scripts/cherrypick.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+set -eu # Exit with nonzero exit code if anything fails
+
+function getTagCommitId() {
+  # get commit id of tag
+  tag_commit=$(curl -s -H "Authorization: token ${GITHUB_BOT_TOKEN}" \
```

Comment:
Getting a commit id by tagname can be done by git. i did the same in tagging script. i think we should avoid the git api whenever possible to make github interchangeable